### PR TITLE
Fix runpath check

### PR DESCRIPTION
**Pull request by frode-aarstad**
**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

## Pre review checklist

- [ ] Added appropriate release note label
- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: src/ert/shared/models/base_run_model.py [hunk 1]
```diff
@@ -535,7 +535,10 @@ def check_if_runpath_exists(self) -> bool:
         idx = [i for i in range(len(active_mask)) if active_mask[i]]
         for i in range(start_iteration, number_of_iterations):
             for j in idx:
-                s = run_path % (j, i)
+                if run_path.count("%d") == 2:
```
Comment (eivindjahren) [inline src/ert/shared/models/base_run_model.py:538]: It is very hard to understand why this is done the way it is. Could you please add some documentation about what is going on.

**Review by eivindjahren — CHANGES_REQUESTED** [review]
Currently, there isn't enough documentation to understand what is going on. Could you please describe what is happening and why we should do it this way? There also needs to be a comment in the code to understand why this is being done.

File: src/ert/shared/models/base_run_model.py (hunk 1 above)
Comment (frode-aarstad) [reply src/ert/shared/models/base_run_model.py:544]: Added a comment to the function to clarify the run_path format

File: src/ert/shared/models/base_run_model.py [hunk 2]
```diff
@@ -525,17 +525,26 @@ def _post_update_data(self, parent_ensemble_id: str, algorithm: str) -> str:
         return update_id
 
     def check_if_runpath_exists(self) -> bool:
+        """
+        Determine if the run_path exists by checking if it contains
+        at least one iteration directory for the iterations in the active mask.
+        The run_path ccan contain one or two %d specifiers ie:
```
Comment (eivindjahren) [inline src/ert/shared/models/base_run_model.py:531]: `can` is misspelled.

Comment (frode-aarstad) [reply src/ert/shared/models/base_run_model.py]: The we also need to update the docs as they state 

`It should contain at least one %d specifier..`

**Review by eivindjahren — APPROVED** [review]

**Pull request merged**